.travis.yml

```diff
@@ -1,15 +1,26 @@
 language: php
 
+sudo: false
```

Comment:
This is deprecated and is the default I think?

Reply:
Erhm, right. I think so. I'll fix this for all libraries one day.